src/webhook/regular_user_validation.py

```diff
@@ -25,22 +31,33 @@ def handle_request():
     return responses.response_invalid()
   
   try:
+    body_dict = request.json['request']
+    username = body_dict['userInfo']['username']
+
+    if body_dict['object'] is None:
+      group_name = body_dict['oldObject']['metadata']['name']
+    else:
+      group_name = body_dict['object']['metadata']['name']
+
     # get the username and decide if it's a special user (SA, kube:admin, etc) or not.. 
     # deny any request if it's not a special user:  kube:*, system:* except system:unauthenticated
+    # ALLOW any request if the group_name is in admin_groups
     # reference: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#request
```

Comment:
Consider moving this comment into `is_request_allowed` itself. (Perhaps even as a docstring.)

Reply:
I'm weak on my python-foo, suggestions for the changes?

Reply:
See below.